docs/DeveloperGuide.md

```diff
@@ -257,27 +257,55 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | medical administrator | quickly add the name, age, address and contact of the patients that require home visits so that          | retrieve them later                                                                                                                             |
+| `* * *`  | medical administrator | quickly add the name, age, address and contact of the nurse/volunteers                                   | retrieve them later for matching                                                                                                                |
+| `* * *`  | medical administrator | add the health conditions (eg heart disease, asthma, paralysed etc) of the patients                      | use this information to match the patient with the respective nurse that has experience in dealing with such disease                            |
+| `* *`    | medical administrator | add a list of diseases that the nurse dealt with before                                                  | give them the appropriate patient for a home visit                                                                                              |
+| `* * *`  | medical administrator | add patients to the existing list of patients that the nurse is going to home visit for the current week | use it to create the schedule for the nurse                                                                                                     |
+| `*`      | medical administrator | add the availability of the nurse                                                                        | use this information to assign the patient to them                                                                                              |
+| `* * *`  | medical administrator | add the date (and time + duration) of home visit appointments for the patient                            | schedule the nurses accordingly                                                                                                                 |
+| `*`      | medical administrator | add next of kin particulars for a patient                                                                | inform them in case of any emergency                                                                                                            |
+| `*`      | medical administrator | add the contact details of the patient's attending physician                                             | liaise with them regarding the patient's treatments and how the patient responds to them                                                        |
+| `*`      | medical administrator | add patients' critical information                                                                       | quickly identify any essential information that needs to be taken note of during scheduling                                                     |
```

Comment:
Maybe this change to nice to have? 
The critical information

Reply:
Are you referring to line 285?

Reply:
yes line 285